layers/+lang/pact/packages.el

```diff
@@ -0,0 +1,50 @@
+;;; packages.el --- Pact Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2018 Kadena LLC
```

Comment:
Note, this line should be:
```emacs-lisp
;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
```

Reply:
Ah, my mistake, I had copy-pasted that originally from the `racket layer and assumed those names were the authors of that layer, not of Spacemacs itself. I'll correct it.